### PR TITLE
Add ChefInspector

### PR DESCRIPTION
**Pull request by Yasumoto**
## Project URL
https://github.com/Yasumoto/ChefInspector

## Category
Development -> Other

## Description
Add an application for querying your Chef Server (both nodes and their attributes).
 
## Why it should be included to `Awesome macOS open source applications `
A good DevOps-y application for macOS.

## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Appropriate language icon(s) added if applicable
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,6 +229,7 @@ Mac app wrapper around Google's stand-alone Android Messenger. ![javascript_icon
 - [j2s](https://github.com/zadr/j2s) - macOS app to convert JSON objects into Swift structs (currently targets Swift 4 and Codable). ![swift_icon]
 - [JSON Mapper](https://github.com/AppCraft-LLC/json-mapper) - Simple macOS app to generate Swift Object Mapper classes from JSON. ![swift_icon]
 - [JSONExport](https://github.com/Ahmed-Ali/JSONExport) - Desktop application for macOS which enables you to export JSON objects as model classes with their associated constructors, utility methods, setters and getters in your favorite language. ![swift_icon]
+- [ChefInspector](https://github.com/Yasumoto/ChefInspector) — Node and Attribute viewer for Chef ![SwiftIcon]
```
[the remark this answers is not in the document]
Comment (nonasoumy) [reply README.md]: Please change from ![SwiftIcon] to ![swift_icon].

[the remark this answers is not in the document]
Comment (Yasumoto) [reply README.md]: Fixed up! Thanks 🙇

**Comment by serhii-londar** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md [hunk 2]
```diff
@@ -246,8 +246,9 @@ Mac app wrapper around Google's stand-alone Android Messenger. ![javascript_icon
 - [Now Desktop](https://github.com/zeit/now-desktop) - Create deployments right from the tray menu. ![javascript_icon]
 - [stts](https://github.com/inket/stts) - macOS app for monitoring the status of cloud services. ![swift_icon]
 
-### Other
+#### Other
```
Comment (serhii-londar) [inline README.md]: Why do you add a '#' symbol? Please remove this change.

Comment (Yasumoto) [reply README.md]: In your TOC this section is a subheading, so looks like it’s misformatted here. Happy to revert if that’s not the case.

Comment (Yasumoto) [reply README.md]: Updated 👍

Comment (serhii-londar) [reply README.md]: > In your TOC this section is a subheading, so looks like it’s misformatted here. Happy to revert if that’s not the case.

I see. That's a good point. I'll fix that. Thank that you mention that!

**Pull request merged**